qemu/tests/luks_convert.py

```diff
@@ -58,3 +58,9 @@ def run(test, params, env):
                                md5_value_to_check=md5_value)
         session.close()
         vm.destroy()
+    logging.debug("Remove the converted img file")
+    try:
+        target.remove()
+    except Exception:
+        print('The source file did not be deleted!')
```

Comment:
We may not need a try here for once we hit an exception, we will get the proper error msg, showing why the file deletion fails